protractor.conf.js

```diff
@@ -29,7 +29,15 @@ exports.config = {
     version: '',
     platform: 'ANY',
     shardTestFiles: true,
-    maxInstances: process.env.CHROME_INSTANCES
+    maxInstances: process.env.CHROME_INSTANCES,
+    download: {
+      prompt_for_download: false
+    },
+    chromeOptions: {
+      prefs: {
+        'download.default_directory':'./web/tmp/downloads'
```

Comment:
@alex-deaconu Do you know if the full directory tree is created if it doesn't exist? Also, I was wondering if it would be better to use `/tmp` for that. Thanks